praetorian_cli/handlers/export.py

```
@@ -12,6 +14,114 @@ def export():
+    result = sdk.post('export/loa', body)
+    if json_out:
+        print_json(result)
+        return
+
+    download_url = result.get('url') or result.get('download_url') or '' if isinstance(result, dict) else ''
+    if download_url and output:
+        import urllib.request
+        resolved = os.path.realpath(os.path.expanduser(output))
+        if os.path.isdir(resolved):
+            resolved = os.path.join(resolved, 'loa.pdf')
+        urllib.request.urlretrieve(download_url, resolved)
```

Ruff (0.15.20)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)